docker.go

```diff
@@ -138,20 +137,28 @@ func (provider *DockerProvider) loadDockerConfig(dockerClient *docker.Client) *C
 			log.Debugf("Filtering disabled container %s", container.Name)
 			return false
 		}
+		_, errRule := provider.getLabel(container, "traefik.frontend.rule")
+		_, errValue := provider.getLabel(container, "traefik.frontend.value")
+
+		if errRule != nil && errValue == nil || errRule == nil && errValue != nil {
```

Comment:
Yeah, you could use inside `getExclusiveLabel` for example :stuck_out_tongue_closed_eyes:.

It's just that I feel it's hard to wrap around the head the condition : `errRule != nil && errValue == nil || errRule == nil && errValue != nil`